src/presets/edgio.ts

```diff
@@ -43,34 +39,20 @@ export const edgio = defineNitroPreset({
       await writeFile(
         resolve(nitro.options.output.dir, "routes.js"),
         `
-import { Router } from '@edgio/core/router'
-import { isProductionBuild } from '@edgio/core/environment'
+import { isProductionBuild } from "@edgio/core/environment";
+import { Router } from "@edgio/core/router";
+import { nitropackRoutes } from "@edgio/nitropack";
```

Comment:
It was intentional that in the initial PR we also avoided extea `@edgio/nitropack` dependency to reduce dependency complexity and looking at [source](https://unpkg.com/browse/@edgio/nitropack@7.5.0/) honestly still it can be all inlined.

If you have strong concerns that need it to be this way, feel free to reach me out in slack or discord (pi0)

Reply:
Based on your comments, I inline implemented the edgio connector.